src/qibocal/protocols/characterization/readout_optimization/resonator_frequency.py

```diff
@@ -208,15 +208,14 @@ def _plot(data: ResonatorFrequencyData, fit: ResonatorFrequencyResults, qubit):
         fitting_report = table_html(
             table_dict(
                 qubit,
-                "Best Resonator Frequency [GHz]",
-                np.round(fit.best_freq[qubit] * HZ_TO_GHZ, 4),
+                "Best Resonator Frequency [Hz]",
+                np.round(fit.best_freq[qubit], 4),
             )
         )
 
     fig.update_layout(
         showlegend=True,
-        uirevision="0",  # ``uirevision`` allows zooming while live plotting
-        xaxis_title="Resonator Frequencies (GHz)",
+        xaxis_title="Resonator Frequencies [GHz]",
```

Comment:
Why is it still in GHz?
```suggestion
        xaxis_title="Resonator Frequencies [Hz]",
```

Reply:
Also in this case GHz is correct because we convert to GHz during the data acquisition.